test/Dialect/FIRRTL/blackbox-reader.mlir

```diff
@@ -1,10 +1,10 @@
 // RUN: split-file %s %t
-// RUN: sed 's?@DIR@?%t?' %t/Foo.mlir.orig > %t/Foo.mlir
-// RUN: circt-opt --pass-pipeline='firrtl.circuit(firrtl-blackbox-reader)' %t/Foo.mlir | FileCheck %t/Foo.mlir
+// RUN: cd %t
+// RUN: circt-opt --pass-pipeline='firrtl.circuit(firrtl-blackbox-reader)' Foo.mlir | FileCheck Foo.mlir
```

Comment:
Nice solution. I was scared of this one. 🙈

Reply:
Yeah, I was struggling with a way to fix it and it turned out to be really easy